### PR TITLE
Feature/add language support

### PR DESCRIPTION
**Pull request by shreesh1993**
Done
- UI fixes for the interactive message template
- UI fixes for the HSM template
- Support for language for the interactive message template

### CODE REVIEW

**Comment by mdshamoon** [comment]
Testing feedbacks: 

1. While sending an interactive message from the chat screen it is showing an error:
image

2. Fields do not get empty or changed if the language is changed. The behavior should be similar to speed sends.

File: src/containers/InteractiveMessage/InteractiveMessage.tsx [hunk 1]
```diff
@@ -131,17 +140,60 @@ export const InteractiveMessage: React.SFC<FlowProps> = ({ match }) => {
...
+    setLabel(labelVal);
+
+    if (languageOptions.length > 0 && languageVal) {
+      const selectedLangauge = languageOptions.find((lang: any) => lang.id === languageVal.id);
+      setTimeout(() => setLanguage(selectedLangauge), 150);
+    }
+
     if (typeValue === LIST) {
       setTitle(data.title);
       setBody(EditorState.createWithContent(WhatsAppToDraftEditor(data.body)));
```
Comment (mdshamoon) [inline src/containers/InteractiveMessage/InteractiveMessage.tsx:199]:
```suggestion
    if (typeValue === LIST || typeValue === QUICK_REPLY) {
      setTitle(data.title);
      setBody(EditorState.createWithContent(WhatsAppToDraftEditor(data.body)));
```

File: src/containers/InteractiveMessage/InteractiveMessage.tsx [hunk 2]
```diff
@@ -151,13 +203,17 @@ export const InteractiveMessage: React.SFC<FlowProps> = ({ match }) => {
     }
 
     if (typeValue === QUICK_REPLY) {
-      setTitle(labelVal);
+      setTitle(data.title);
       setBody(EditorState.createWithContent(WhatsAppToDraftEditor(data.body)));
```
Comment (mdshamoon) [inline src/containers/InteractiveMessage/InteractiveMessage.tsx:207]:
```suggestion

```
Remove

File: src/containers/InteractiveMessage/InteractiveOptions/QuickReplyTemplate.tsx [hunk 3]
```diff
@@ -50,13 +50,18 @@ export const QuickReplyTemplate: React.SFC<QuickReplyTemplateProps> = (props) =>
             onChange={(e: any) => handleInputChange(e, 'value')}
             value={defaultValue}
             className={styles.TextField}
+            InputProps={{
+              endAdornment:
+                inputFields.length > 1 ? (
+                  <CrossIcon className={styles.RemoveIcon} title="Remove" onClick={onRemoveClick} />
+                ) : null,
```
Comment (mdshamoon) [inline src/containers/InteractiveMessage/InteractiveOptions/QuickReplyTemplate.tsx:57]:
```suggestion
                inputFields.length > 1 && (
                  <CrossIcon className={styles.RemoveIcon} title="Remove" onClick={onRemoveClick} />
                ) ,
```

Let's assign this and adornment as a const above and use it here. WIll be better for code readability.

**Comment by shreesh1993** [comment]
> Testing feedbacks:
> 
> 1. While sending an interactive message from the chat screen it is showing an error:
>    image

Could you please check backend branch, I think that is not yet on master. please check this [branch](https://github.com/glific/glific/pull/1667)

**Review by mdshamoon — APPROVED** [review]

**Pull request merged**